### ISSUE TITLE
Rate Limit impacting non-captcha pages

### ISSUE THREAD

**Issue by MarcPinnell** [datamweb/CodeIgniter-Multi-Captcha#8]
My application has both an admin and public side. I have installed the code for the captcha protection on the three public facing forms and it appears to be working.
I just received a call from my client. She is working in the admin and keeps getting the Rate Limit warning page as she moves around the admin pages. None of which have any captcha code. Is there a way to limit the captcha checking to a path?
In other words I would want it to work on https://domain.com/ and sub pages, but not work on https://domain.com/admin and those pages. The public and admin sides of the app have different controllers.

**Comment by datamweb** [comment]
Hi, try the following way.

```diff
 public array $globals = [
'before' => [
+ 'rate_limit_by_captcha' => ['except' => ['admin*']]
],
```
```diff
public array $filters = [
- 'rate_limit_by_captcha' => ['before' => ['/*']]
+ //'rate_limit_by_captcha' => ['before' => ['/*']]
];
```
I haven't checked this, please let me know if it fixes the problem.

**Comment by MarcPinnell** [comment]
Thanks
